templates/tilecloudchain-deployment.yaml

```diff
@@ -0,0 +1,64 @@
+{{- if .Values.tilecloudchain.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "common.fullname" ( dict "root" . "service" .Values.tilecloudchain ) }}
+  labels: {{ include "common.labels" ( dict "root" . "service" .Values.tilecloudchain ) | nindent 4 }}
+  annotations: {{- toYaml .Values.tilecloudchain.annotations | nindent 4 }}
+spec:
+  replicas: {{ .Values.config.replicaCount }}
+  strategy:
+    type: RollingUpdate
+  selector:
+    matchLabels: {{ include "common.selectorLabels" ( dict "root" . "service" .Values.tilecloudchain ) | nindent 6 }}
+  template:
+    metadata:
+      labels: {{ include "common.labels" ( dict "root" . "service" .Values.tilecloudchain ) | nindent 8 }}
+    spec:
+      {{- include "common.podConfig" ( dict "root" . "service" .Values.tilecloudchain ) | nindent 6 }}
+      containers:
+        - name: config
+          {{- include "common.containerConfig" ( dict "root" . "container" .Values.tilecloudchain.config ) | nindent 10 }}
+          args: ['shared-config-slave']
+          readinessProbe: &configProbe
+            exec:
+              command:
+                - scm-is-ready
+            initialDelaySeconds: 30
+            periodSeconds: 30
+            failureThreshold: 5
+          livenessProbe:
+            <<: *configProbe
+            failureThreshold: 50
+          volumeMounts:
+            - name: configs
+              mountPath: /master_config
+        - name: tilecloudchain
+          {{- include "common.containerConfig" ( dict "root" . "container" .Values.tilecloudchain ) | nindent 10 }}
+          volumeMounts:
+            - name: configs
+              mountPath: /etc/tilecloudchain-configs
+            - name: hosts
+              mountPath: /etc/tilecloudchain-hosts
+            - name: mainconfig
+              mountPath: /etc/tilecloudchain-main
+          ports:
+            - name: http
+              containerPort: 8080
+              protocol: TCP
+          readinessProbe:
+            httpGet:
+              path: /tiles/c2c/health_check
+              port: http
+            initialDelaySeconds: 5
+            timeoutSeconds: 10
```

Comment:
I would add a liveness probe for tilecloudchain, the same as the readiness should be ok (i.e. HTTP GET `/tiles/c2c/health_check` )

Reply:
Done